pkg/katautils/config.go

```diff
@@ -419,6 +421,20 @@ func (h hypervisor) guestHookPath() string {
 	return h.GuestHookPath
 }
 
+func (h hypervisor) rxRateLimiter() (int64, error) {
+	if h.RxRateLimiter < 0 {
+		return 0, fmt.Errorf("Invalid Firecracker Rx Rate Limiter value %v", h.RxRateLimiter)
```

Comment:
I see this option only applies to FC. However, this is generic code for "any hypervisor", so I'd be tempted to remove "Firecracker" from the error messages here. That would protect against the potentially very confusing scenario where we add the feature to other hypervisors and the user gets an error relating to a hypervisor they aren't using ;)

Reply:
thanks. I'll remove "Firecracker" from the error messages here. ;)